trpcultivate_germplasm/src/Plugin/TripalImporter/GermplasmCrossImporter.php

```diff
@@ -0,0 +1,963 @@
+    // input file.
+    $instance->setHeaders($this->headers);
+    // Configure the expected number of columns and set it to be strict.
+    $num_columns = count($this->headers);
+    $instance->setExpectedColumns($num_columns, TRUE);
+    $validators['header-row']['valid_header'] = $instance;
+
+    // -----------------------------------------------------
+    // Data Row Level
+    // - All data row cells in columns 0-5 are not empty
+    $instance = $this->service_validatorPluginManager->createInstance('empty_cell');
+    $indices = [
+      $header_index['Year'],
+      $header_index['Season'],
+      $header_index['Cross Number'],
+      $header_index['Uniquename'],
+      $header_index['Maternal Parent'],
+      $header_index['Paternal Parent'],
+      $header_index['Cross Type'],
+    ];
+    $instance->setIndices($indices);
+    $validators['data-row']['empty_cell'] = $instance;
+
+    // - The column 'Season' is one of: Winter, Spring, Summer, Fall
+    $instance = $this->service_validatorPluginManager->createInstance('value_in_list');
+    $instance->setIndices([$header_index['Season']]);
+    $instance->setValidValues([
+      'Winter',
+      'Spring',
+      'Summer',
+      'Fall',
+    ]);
+    $validators['data-row']['valid_season'] = $instance;
+
+    // - Maternal Parent and Paternal Parent cells exist in the database.
+    $instance = $this->service_validatorPluginManager->createInstance('germplasm_name_exists');
+    // Set the logger since this validator uses a setter (setOrganismID)
+    // which may log messages.
+    $instance->setLogger($this->logger);
+    $indices = [
+      $header_index['Maternal Parent'],
+      $header_index['Paternal Parent'],
+    ];
+    $instance->setIndices($indices);
+    $organism_id = $form_values['organism'];
+    $instance->setOrganismID($organism_id);
+    $validators['data-row']['germplasm_name_exists'] = $instance;
+    return $validators;
+  }
+
+  /**
+   * {@inheritDoc}
+   */
+  public function form($form, &$form_state) {
+    // Always call the parent form to ensure Chado is handled properly.
+    $form = parent::form($form, $form_state);
+
+    // Validation result.
+    $storage = $form_state->getStorage();
+
+    // Full validation result summary.
+    if (isset($storage[$this->validation_result])) {
+      $validation_result = $storage[$this->validation_result];
+
+      $form['validation_result'] = [
+        '#type' => 'inline_template',
+        '#theme' => 'validation_result_window',
+        '#data' => [
+          'validation_result' => $validation_result,
+        ],
+        '#weight' => -100,
+      ];
+    }
+
+    // Field Organism:
+    // Prepare select options with only active organisms.
+    $all_organisms = chado_get_organism_select_options();
+
+    // If there is only one organism, it should be the default.
+    $default_organism = 0;
+    if ($all_organisms && count($all_organisms) == 1) {
+      $default_organism = reset($all_organisms);
```

Comment:
```suggestion
      $default_organism = array_keys($all_organisms)[0];
```
The Organism field does not default to an organism. Try this..

Reply:
Thanks for catching this Reynold! Addressed in commit https://github.com/TripalCultivate/TripalCultivate-Germplasm/pull/33/commits/4f21ec27404dd8611a71562524425cb31dc42035